src/collectd.conf.pod

```diff
@@ -540,6 +540,7 @@ B<Synopsis:>
    # Send values to an AMQP broker
    <Publish "some_name">
      Host "localhost"
+#    Host "amqp1.example.com" "amqp2.example.com" "amqp3.example.com"
```

Comment:
I find this comment a bit distracting. How about amending the existing line, e.g. to:

```
Host "localhost"
Host "fallback-amqp.example.com"
```

Reply:
good idea. pushed